cmake/modules/platform/toolcfg/xlc.cmake

```diff
@@ -32,6 +32,12 @@ if(OMR_HOST_ARCH STREQUAL "ppc")
 
 	list(APPEND OMR_PLATFORM_CXX_COMPILE_OPTIONS -qlanglvl=extended0x)
 
+	if(NOT OMR_ENV_LITTLE_ENDIAN)
+		# add -qdebug=NETHUNK for BE platforms to C/CXX flags
+		list(APPEND OMR_PLATFORM_COMPILE_OPTIONS -qdebug=NETHUNK)
+		list(APPEND OMR_PLATFORM_CXX_COMPILE_OPTIONS -qdebug=NETHUNK)
```

Comment:
`OMR_PLATFORM_COMPILE_OPTIONS` applies to [both C and C++](https://github.com/eclipse/omr/blob/master/cmake/modules/OmrPlatform.cmake#L62-L63), so there's no need to add this option to `OMR_PLATFORM_CXX_COMPILE_OPTIONS`.

```suggestion
```

Reply:
removed option from OMR_PLATFORM_CXX_COMPILE_OPTIONS